reference/tools/tiup/cluster.md

```diff
@@ -0,0 +1,565 @@
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster start prod-cluster
+```
+
+If you forget the name of your cluster, view the cluster list by running `tiup cluster list`.
+
+## Check the cluster status
+
+TiUP provides the `tiup cluster display` command to view the status of each component in the cluster. With this command, you don't have to log in to each machine separately to see the component status. The usage of the command is as follows:
+
+{{< copyable "shell-root" >}}
+
+```bash
+tiup cluster display prod-cluster
+```
+
+```
+Starting /root/.tiup/components/cluster/v0.4.5/cluster display prod-cluster
+TiDB Cluster: prod-cluster
+TiDB Version: v3.0.12
+ID                  Role        Host          Ports        Status     Data Dir              Deploy Dir
+--                  ----        ----          -----        ------     --------              ----------
+172.16.5.134:3000   grafana     172.16.5.134  3000         Up         -                     deploy/grafana-3000
+172.16.5.134:2379   pd          172.16.5.134  2379/2380    Healthy|L  data/pd-2379          deploy/pd-2379
+172.16.5.139:2379   pd          172.16.5.139  2379/2380    Healthy    data/pd-2379          deploy/pd-2379
+172.16.5.140:2379   pd          172.16.5.140  2379/2380    Healthy    data/pd-2379          deploy/pd-2379
+172.16.5.134:9090   prometheus  172.16.5.134  9090         Up         data/prometheus-9090  deploy/prometheus-9090
+172.16.5.134:4000   tidb        172.16.5.134  4000/10080   Up         -                     deploy/tidb-4000
+172.16.5.139:4000   tidb        172.16.5.139  4000/10080   Up         -                     deploy/tidb-4000
+172.16.5.140:4000   tidb        172.16.5.140  4000/10080   Up         -                     deploy/tidb-4000
+172.16.5.134:20160  tikv        172.16.5.134  20160/20180  Up         data/tikv-20160       deploy/tikv-20160
+172.16.5.139:20160  tikv        172.16.5.139  20160/20180  Up         data/tikv-20160       deploy/tikv-20160
+172.16.5.140:20160  tikv        172.16.5.140  20160/20180  Up         data/tikv-20160       deploy/tikv-20160
+```
+
+For ordinary components, the `Status` column shows `Up` or `Down`, which indicates whether the service is running normally.
+
+For the PD component, the `Status` column shows `Healthy` or `Down`, and sometimes `|L` if the PD node is a Leader.
+
+## Scale in a node
+
+> **Note:**
+>
+> This section describes only the example syntax of the scale-in command. For detailed steps of online scaling, refer to [Scale the TiDB Cluster Using TiUP](/how-to/scale/with-tiup.md).
+
+Scaling in a node means taking the node offline. This operation removes the node from the cluster and deletes the related data files.
+
+Because the offline process of the TiKV and TiDB Binlog components is asynchronous (which requires removing the node through API), and the process takes a long time to see if the node is successfully taken offline, special treatment is given to the TiKV and TiDB Binlog components.
```

Comment:
```suggestion
Because the offline process of the TiKV and TiDB Binlog components is asynchronous (which requires removing the node through API), and the process takes a long time (which requires constant observation on whether the node is successfully taken offline), special treatment is given to the TiKV and TiDB Binlog components.
```